.rubocop.yml

```diff
@@ -6,3 +6,6 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
     - 'spec/datadog/appsec/waf_spec.rb'
+
+Style/HashSyntax:
+  EnforcedShorthandSyntax: never
```

Comment:
Minor: May be worth leaving here a note on why this was disabled, so it's clear in the future when we can remove it (ahaha although I can't imagine the future where we drop support for < 3.1 seems so far away)

```suggestion
# Requires Ruby 3.1
Style/HashSyntax:
  EnforcedShorthandSyntax: never
```